components/clicktime/actions/create-user/create-user.mjs

```diff
@@ -0,0 +1,88 @@
+import app from "../../clicktime.app.mjs";
+
+export default {
+  key: "clicktime-create-user",
+  name: "Create User",
+  description: "Create an User on ClickTime. [See the documentation](https://developer.clicktime.com/docs/api/#/operations/Users/CreateManagedUser)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    billingRate: {
```

Comment:
According to the [documentation](https://developer.clicktime.com/docs/api/#/Users/CreateManagedUser), `email` and `name` are the only required fields, so the others should be optional.